src/UiControl.cc

```diff
@@ -1,14 +1,39 @@
-#include "../ui.h"
-#include "ui-node.h"
+#include <map>
 #include "nbind/api.h"
-#include "nbind/nbind.h"
+#include "control.h"
+#include "ui.h"
+
+std::map<uiControl *, UiControl *> controlsMap;
```

Comment:
This map is used to retrieve UiControl instances given their libui control handle.
UiArea has a similar map, that can be now removed.